shopping-cart/index.html

```diff
@@ -11,51 +11,54 @@
 <body>
     <header>America's Top Public Companies - Cart</header>
     <script type="text/javascript" src="../common/nav.js"></script>
-    <table>
-        <thead>
-            <tr>
-                <th class="align-left">Company Stock</th>
-                <th>Qty</th>
-                <th>Price</th>
-                <th>Total</th>
-            </tr>
-        </thead>
-        <tbody>
-            <!-- <tr>
-                <td>apple</td>
-                <td>4</td>
-                <td>$1.00</td>
-                <td>$4.00</td>
-            </tr>
-            <tr>
-                <td>apple</td>
-                <td>4</td>
-                <td>$1.00</td>
-                <td>$4.00</td>
-            </tr>
-            <tr>
-                <td>apple</td>
-                <td>4</td>
-                <td>$1.00</td>
-                <td>$4.00</td>
-            </tr>
-            <tr>
-                <td>apple</td>
-                <td>4</td>
-                <td>$1.00</td>
-                <td>$4.00</td>
-            </tr> -->
-        </tbody>
-        <tfoot>
-            <tr>
-                <th></th>
-                <th></th>
-                <th>Order Total:</th>
-                <th id="order-total-cell"></th>
-            </tr>
-        </tfoot>
+    <div id="table-div">
+        <table>
+            <thead>
+                <tr>
+                    <th class="align-left">Company Stock</th>
+                    <th>Qty</th>
+                    <th>Price</th>
+                    <th>Total</th>
+                </tr>
+            </thead>
+            <tbody>
+                <!-- <tr>
```

Comment:
You can delete this commented out code. You want to keep your code clean and only keep the code you are using/that works.